recipes/prismatic/meta.yaml

```diff
@@ -115,7 +115,7 @@ outputs:
 
     requirements:
       build:
-        - python 3.*
```

Comment:
This commit should be reverted and
```
build:
  skip: True  # [win or py2k]
```
should be used instead of adding a `3.*` version requirement on Python.

Reply:
Thanks for the reminder (as I made this mistake again in my new code) - I've corrected this now.